dessia_api_client/__init__.py

```diff
@@ -2,5 +2,5 @@
 # -*- coding: utf-8 -*-
 
 import pkg_resources
-from .core import *
-__version__ = pkg_resources.require("dessia_api_client")[0].version
\ No newline at end of file
+from . import *
+__version__ = pkg_resources.require("dessia_api_client")[0].version
```

Comment:
I didn't understand how does this work, I left it as is.
in local I always comment the line: 
`
__version__ = pkg_resources.require("dessia_api_client")[0].version
`